app/views/CreditAndRefunds.scala.html

```diff
@@ -89,21 +89,21 @@
 }
 
 @getCreditsSummaryLink = {
-    @if(maybeCreditChargesHead._2.isMFACredit) {
+    @if(maybeCreditChargesHead._2.getCreditType == Some("MFA Credit")) {
```

Comment:
these checks should be localised on the different level, so we already should check against strong types here